### ISSUE TITLE
Ask Cody: generate release notes

### ISSUE THREAD

**Issue by malomarrec** [sourcegraph/sourcegraph#50527]

**Comment by deepak2431** [comment]
@malomarrec For now, we have a recipe asking Cody to summarize recent code changes based on the commit history. We can use this idea to create recipes for the release notes.
The question here is:
1. How many previous commit summaries should we use to generate this?
2. May be we ask the user to select the date range for the commits or provide any other details that can be used to generate precise release notes?

@mrnugget Putting this to your attention if you think this recipe should be added.

You can see this attached image, where I asked Cody to generate the release note based on the recent 5 code commits.

![iScreen Shoter - Code - 230426233925](https://user-images.githubusercontent.com/44617923/234665625-ca2189c3-597e-4c3e-9b3f-39b20c31bf75.jpg)

**Comment by mrnugget** [comment]
@deepak2431 I think ideally we'd ask the user for a timeframe for which to generate release notes. Maybe offer `tags` as a starting point too (i.e. "Release notes for changes since tag `0.7`").

Is what you added a standalone recipe or does it work on conjunction with the summary recipe?

**Comment by deepak2431** [comment]
@mrnugget Thanks for your input. Based on it, I think we can ask for the time frame and the tags as the input. If the tag is not found for the repository, or if the projects don't have tags, then we can use the commits of the given timeframe to generate the release notes.

I haven't added it as a recipe yet. I was testing the idea of using the commits summary to generate the release notes, which I think can be used. For me, it would be better to keep it as a separate recipe.

I just wanted to confirm once with @malomarrec, are the cody-recipes issues open to work for now? If yes, then I can make a PR for this.

**Comment by mrnugget** [comment]
Yeah, go for it! We're super busy and we don't get to all the recipes, so contributions are welcome :)

**Comment by deepak2431** [comment]
Great, will work on this!
